### PR TITLE
Add Android In-App Review

### PR DESCRIPTION
**Pull request by jamesmontemagno**
Also add macOS :)

Please take a moment to fill out the following:

Fixes # .

Changes Proposed in this pull request:
-
-
-

### CODE REVIEW

File: src/StoreReviewTest/StoreReviewTest/MainPage.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    public partial class MainPage : ContentPage
+    {
+        public MainPage()
+        {
+            InitializeComponent();
+        }
+
+		async void Button_Clicked(object sender, EventArgs e)
+		{
+			await CrossStoreReview.Current.RequestReview(true);
```
Comment (saamerm) [inline src/StoreReviewTest/StoreReviewTest/MainPage.xaml.cs:21]: The PR looks good to me. @jamesmontemagno 
Should we add a pre-compile check here so users know to set it to false for release?
```
bool isTestMode = false;
#if DEBUG
isTestMode = true;
#endif 
await CrossStoreReview.Current.RequestReview(isTestMode);
```
Also, let me know if you would like me to test it too

**Comment by saamerm** [comment]
You can also slide in the changes to the README.md url https://devblogs.microsoft.com/xamarin/requesting-reviews-ios-10-3s-skstorereviewcontroller/ from the older xamarin website, or we can update the README in the another PR

Also, let me know if you would like me to test it too

**Comment by jamesmontemagno** [comment]
Are you able to grab the code and verify on your test app? I don't have it in the store so can't verify

**Comment by saamerm** [comment]
> Are you able to grab the code and verify on your test app? I don't have it in the store so can't verify

Sure. I have to change this to `false` though for it to work. Will let you know as soon as I get a chance
```
await CrossStoreReview.Current.RequestReview(false); 
```

**Comment by jamesmontemagno** [comment]
You can also grab a nuget from : https://dev.azure.com/jamesmontemagno/Plugins%20for%20Xamarin/_build/results?buildId=1862&view=artifacts&type=publishedArtifacts

**Comment by saamerm** [comment]
Alright @jamesmontemagno ! 
I just tested it and it looks good. I tried canceling it and that worked as well!
You're good to go 👍

**Review by saamerm — APPROVED** [review]
Alright @jamesmontemagno ! 
I just tested it by getting the nuget package from the[ URL you shared](https://dev.azure.com/jamesmontemagno/Plugins%20for%20Xamarin/_build/results?buildId=1862&view=artifacts&type=publishedArtifacts), installed it in my app and then uploaded that to the Play Store "Internal App Sharing" and it looks good. 

I tried canceling it and that worked as well!
You're good to go 👍

**Review by saamerm — APPROVED** [review]

**Comment by jamesmontemagno** [comment]
Lovely! will pull in :)

**Pull request merged**